src/test/java/com/jumani/rutaseg/controller/ConsigneeControllerTest.java

```diff
@@ -0,0 +1,123 @@
+package com.jumani.rutaseg.controller;
+
+import com.jumani.rutaseg.domain.Consignee;
+import com.jumani.rutaseg.domain.Client;
+import com.jumani.rutaseg.dto.request.ConsigneeRequest;
+import com.jumani.rutaseg.dto.response.SessionInfo;
+import com.jumani.rutaseg.exception.ValidationException;
+import com.jumani.rutaseg.repository.client.ClientRepository;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class ConsigneeControllerTest {
+
+    @Mock
+    private ClientRepository clientRepository;
+
+    @InjectMocks
+    private ConsigneeController consigneeController;
+
+    @Test
+    void createConsignee_Success() {
+        Long clientId = 1L;
+        SessionInfo sessionInfo = new SessionInfo(1L, true);
+        ConsigneeRequest consigneeRequest = new ConsigneeRequest();
+        consigneeRequest.setName("Name");
+        consigneeRequest.setCuit(123456789);
+
+        Client client = mock(Client.class);
+        when(clientRepository.findById(clientId)).thenReturn(Optional.of(client));
+
+        ResponseEntity<Consignee> response = consigneeController.createConsignee(clientId, sessionInfo, consigneeRequest);
+
+        assertEquals(HttpStatus.CREATED, response.getStatusCode());
+        assertNotNull(response.getBody());
+    }
+
+    @Test
+    void createConsignee_Failure_InvalidUser() {
```

Comment:
te parece agregar un test donde la sesion no es admin y no coincide el userId del client? Debería salir por not found también